gravitee-apim-console-webui/src/entities/management-api-v2/api/v4/healthCheck.fixture.ts

```diff
@@ -32,30 +32,21 @@ export function fakeApiHealthResponseTimeOvertime(attribute?: Partial<ApiHealthR
   };
 }
 
-export function fakeApiHealthAvailability(attribute?: Partial<ApiAvailability>): ApiAvailability {
-  const base: ApiAvailability = {
-    global: 0.9876,
-    group: {
-      example: 100,
```

Comment:
you can just replace "example field" with your someSampleGroup, no need to modify function as a whole.

Reply:
I can't 😞 That solution you propose doesn't work in this case. I started with it and had to find another way of creating a dictionary to make this fixture work.